workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/common/I18n/Language.json

```diff
@@ -213,6 +213,7 @@
 	"if-you-need-more-information-about-the-availability-of-your-x-activation-keys-please-ask-one-of-your-administrator-team-members-to-update-your-permissions-so-you-can-contact-liferay-support-alternatively-team-members-with-administrator-or-requester-role-can-submit-a-support-ticket-on-your-behalf": " If you need more information about the availability of your {0} activation keys, please ask one of your Administrator team members to update your permissions, so you can contact Liferay Support. Alternatively, team members with Administrator or Requester role can submit a support ticket on your behalf.",
 	"in-progress": "In Progress",
 	"inactive": "Inactive",
+	"incident-contact": "Incident Contact",
```

Comment:
is this needed? it looks like you're pushing the already translated value and not a language key in the .js

Reply:
Yes, this is necessary because the roles use the translation key, otherwise it would be “incident-contact” in the team members table.

Reply:
ah ok I see, converting back to kebabcase is unusual